app/views/order_items/_order_item.html.erb

```diff
@@ -5,6 +5,17 @@
         <%= image_tag product.avatar %>
       </div>
       <div class="caption">
+        <h6>
+          <% if product.calories != nil %>
+            <%= content_tag( :small) do
+              concat( product.calories)
+              concat ' kcal'
+              end
+            %>
+          <% else %>
```

Comment:
Is it necessary to put an empty thing here? I don't see the point as you obtain an empty `h6` now